extensions/amp-story/1.0/amp-story.js

```diff
@@ -483,6 +484,7 @@ export class AmpStory extends AMP.BaseElement {
       Action.TOGGLE_PAUSED,
       this.pausedStateToRestore_
     );
+    this.playBackgroundAudio_();
```

Comment:
Do we want to check if pausedToRestoreState is true? So if you pause, then lose focus, then gain focus, it doesn't start playing the audio even if the story is still paused

Reply:
That is the intended behavior yes!